app/controllers/static_pages_controller.rb

```diff
@@ -6,7 +6,7 @@ def search
       redirect_to(root_path, alert: 'Empty field!') && return
     else
       @parameter = params[:search].downcase
-      @results = Topic.all.where('lower(name) LIKE :search', search: "%#{@parameter}%")
+      @results = Cheatsheet.all.where('lower(topic) LIKE :search', search: "%#{@parameter}%")
```

Comment:
Now I think, that this is going to give you cheat sheets for all users (note that it's not scoped by user). Is that OK or were you going for searching for cheat sheets for the current user?

Reply:
I think search is a general search for looking up all cheatsheets 🤔 could be wrong though 😂 . @janessatran where do you use search?

Reply:
I want to have the search bar be a general search, but I should maybe consider adding another parameter for specifying public and private so that the user can specify if they want their cheatsheet to be viewed by everyone. @botanical i plan to use the search on the home page of the app :)

Reply:
I was actually thinking of public/private, so I'm for that anyway :)